psamm/datasource/sbml.py

```diff
@@ -709,6 +709,21 @@ def _add_flux_bounds(self, r_id, model, flux_limits, equation):
             upper_str = text_type(flux_limits[r_id][1])
         return lower_str, upper_str
 
+    def _add_fbc_objective(self, m_tag, fbc, obj_id):
```

Comment:
Add docstrings to these methods. You can also try to improve the parameter names a bit. Maybe `model_tag` is better than `m_tag`? Maybe you don't have to pass `fbc`?